tracing-opentelemetry/src/main/java/org/axonframework/tracing/opentelemetry/OpenTelemetrySpan.java

```diff
@@ -120,4 +120,12 @@ public Span recordException(Throwable t) {
         span.setStatus(StatusCode.ERROR, t.getMessage());
         return this;
     }
+
+    @Override
+    public Span addAttribute(String key, String value) {
+        if(this.span == null) {
+            spanBuilder.setAttribute(key, value);
+        }
```

Comment:
I would kind of expect `this.span.setAttribute(key, value)` if it's not null)? But my understanding might be lacking.

Reply:
You are right. I didn't think this method existed, but it does! Adjusting.